test/check-machines-create

```diff
@@ -1489,7 +1489,7 @@ vnc_password= "{vnc_passwd}"
                     b.mouse("#create-and-edit", "mouseenter")
                     b.wait_visible("#create-and-edit-disabled-tooltip")
                 else:
-                    b.wait_visible("#create-and-run[aria-disabled=false]")
+                    b.wait_visible("#create-and-run:not([aria-disabled=false])")
```

Comment:
This doesn't make much sense IMHO. I suppose the intention was to check that the button is not disabled (i.e. is clickable). But then it should check `:not([aria-disabled=true])`

Reply:
Right, mis-copy pasta, thanks for spotting